resources/language/resource.language.nl_nl/strings.po

```diff
@@ -197,42 +197,44 @@ msgctxt "#30416"
 msgid "We could not find any programs that were followed.\n\nEither right-click on a program or an episode to follow a program, or follow a program on the VRT NU website."
 msgstr "We konden geen programma's vonden die je volgt.\n\nJe kan een programma volgen door rechts te klikken op een programma of aflevering, of om ze op de VRT NU website to volgen."
 
+
+### SETTINGS
 msgctxt "#30800"
+msgid "Credentials"
+msgstr "Inloggegevens"
+
+msgctxt "#30801"
+msgid "Email address"
+msgstr "E-mailadres"
+
+msgctxt "#30803"
+msgid "Password"
+msgstr "Wachtwoord"
+
+msgctxt "#30820"
 msgid "Interface"
 msgstr "Interface"
 
-msgctxt "#30801"
+msgctxt "#30821"
 msgid "Enable My programs"
 msgstr "Toon Mijn TV programma's"
```

Comment:
Please use valid Dutch spelling: https://vrttaal.net/taaladvies-spelling/tv

Reply:
Well, I opted to use the same as the VRT NU website, where they use TV-gids.
(In fact this should have a dash)

Reply:
Fixed.